library/ZendService/Twitter/Response.php

```diff
@@ -152,4 +160,52 @@ public function toValue()
     {
         return $this->jsonBody;
     }
+
+    /**
+     * Retun the array of response rate info
+     *
+     * @return array
+     */
+    public function getRateLimit()
+    {
+        return $this->rateLimit;
+    }
+
+    /**
+     * Populates the object with info. This can possibly called from the
+     * constructor, or it can be called later.
+     *
+     */
+    public function populate($httpResponse = null)
+    {
+
+        if (is_null($httpResponse)) {
+            $httpResponse = $this->httpResponse;
+        }
+
+        $this->httpResponse = $httpResponse;
+        $this->rawBody      = $httpResponse->getBody();
+
+        $headers = $this->httpResponse->getHeaders();
+
+        if (! is_null($headers)) {
+            $headers = $headers->toArray();
```

Comment:
Hi! :)

I can see where that may be true. I can change this to set a holding variable and then the final ->toArray() is stored in $headers. Personally, I find that to be more confusing as you are introducing an extra variable. However, this is a personal style preference. I don't find it any clearer to read and it would execute almost exactly the same. 

Cheers!
=C=